Add missing internationalization in app/Sketch.java

joelmoniz opened the pull request:
This PR adds in a couple of key-value pairs into PDE.properties file from the app/Sketch.java that were hard-coded so far. This should complete the internationalization of app/Sketch.java.

benfry commented:
Don't all those quotes need to be escaped, the way they are in the rest of the file?

joelmoniz commented:
I had actually escaped them initially, but then removed the escapes when I realized that most of the file actually contained quotes that weren't escaped. I tested this out as well, and it seems that strings read in from a text file don't need anything to be escaped. I checked this out again just now, and surprisingly, strings with escaped characters when read from a file print the escaping slash as well, so something like `The file \"%s\" already exists.` actually shows up as `The file \"sketch_150618a\" already exists.`, while `The file "%s" already exists.` shows as expected: `The file "sketch_150618a" already exists.`  

NB: I've also updated the strings that had escaping backslashes in them and removed them.

benfry commented:
Ok, thanks for checking.
